src/features/collaborator-avatars/__tests__/CollaboratorAvatarItem.test.js

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
 
-import CollaboratorAvatarItem from '../CollaboratorAvatarItem';
+import { CollaboratorAvatarItemBase as CollaboratorAvatarItem } from '../CollaboratorAvatarItem';
+
+const intl = {
+    formatMessage: message => message.defaultMessage,
```

Comment:
Should we use jest?
```
const intl = { formatMessage: jest.fn().mockImplementation((message) => message.defaultMessage) };
```

Reply:
Trevor had suggested this approach in a different PR, is there a reason why would we go with jest here instead?

https://github.com/box/box-ui-elements/pull/3189#discussion_r1052819182

Reply:
I suppose the reason would be consistency, I see it everywhere else in the code use jest. But if manual mocking works, that's ok too!

Reply:
@megansmith-box yeah that might've been a good idea to use jest. I saw some other tests do manual mocking so I didn't really think of using `mockImplementation` but maybe we could migrate all those to use jest in the future